### ISSUE TITLE
Change Value Curve option "Timing Track Toggle" to "Timing Track Toggle Non-latching" or similar for clarity

### ISSUE THREAD

**Issue by SopherConklin** [xLightsSequencer/xLights#4801]
In the world outside of xLights, "Toggle" means to permanently switch from 1 condition to another.
In xLights value curve, "Toggle" means to temporarily switch to 1 condition for the length of the Timing Track Interval.
This is misleading and confusing.   A minor point but also a minor fix.  Something like Timing Track Interval Toggle,  Timing Track Temporary Toggle, Non-latching Timing Track Toggle would more clearly communicate to the user what it actually does.   Yes, we can experiment to find out how it works, but why not make it clearer.

**Comment by keithsw1111** [comment]
No.

**Comment by SopherConklin** [comment]
You say "No."
I hear "We like having confusing and obtuse labels for things in xLights.
It puts the newbies in their place and makes us oldtimers feel smarter."

**Comment by dkulp** [comment]
I agree with Keith.    To me, none of your suggestions are significantly better than the original.  Changing it is NOT a "minor fix" as there is significant amount of work to update docs, adjust all loaded sequences, handle imports/mappings, etc...  And in the end, sequences using the "new" name would then not load properly into older xLights (although that is a very minimal concern).  Basically, not worth the hassle and work for an almost insignificant difference.

**Comment by SopherConklin** [comment]
Thank you for that reasonable response.
